packages/hooks/src/useResizeObserver.mdx

```diff
@@ -22,10 +22,8 @@ import { useResizeObserver } from "@jobber/hooks";
 
 <Playground>
   {() => {
-    const [
-      ref,
-      { width, exactWidth, exactHeight, height },
-    ] = useResizeObserver();
+    const [ref, { width, exactWidth, exactHeight, height }] =
+      useResizeObserver();
```

Comment:
this is very interesting. still shows up as

![image](https://user-images.githubusercontent.com/15986172/134221874-8cae678e-a7b6-4615-a82c-4293655dbbef.png)

but on the mdx, it's not formatted like that. still works tho